video/out/d3d11/ra_d3d11.c

```diff
@@ -1599,6 +1599,7 @@ static void save_cached_program(struct ra *ra, struct ra_renderpass *pass,
     };
     memcpy(header.magic, cache_magic, sizeof(header.magic));
     strncpy(header.compiler, spirv->name, sizeof(header.compiler) - 1);
+    header.compiler[sizeof(header.compiler) - 1] = '\0';
```

Comment:
Shouldn't be needed after https://github.com/mpv-player/mpv/pull/12420/commits/5ab3d03c7103ddf04c5ef0d0be25ba800e2f509a